src/display.rs

```diff
@@ -278,8 +278,10 @@ impl Display {
         // Clear dirty flag
         terminal.dirty = !terminal.visual_bell.completed();
 
-        if let Some(title) = terminal.get_next_title() {
-            self.window.set_title(&title);
+        if config.dynamic_title() {
```

Comment:
Totally bikeshedding here but, this led me to https://github.com/rust-lang/rfcs/issues/929. You *could* change this to:

```rust
if let (Some(title), true) = (terminal.get_next_title(), config.dynamic_title()) {
    // ...
}
```

But I probably wouldn't lol.

Reply:
I wouldn't either, it looks interesting indeed, but I think it's less clear to me than the current code. Anything else I should change?